integrations/os2sync/os2mo.py

```diff
@@ -141,7 +146,10 @@ def addresses_to_user(user, addresses):
 
 
 def engagements_to_user(user, engagements, allowed_unitids):
-    for e in engagements:
+    for e in sorted(
+        engagements,
+        key=lambda e: e["job_function"]["name"] + e["uuid"]
```

Comment:
Denne her sorteringsnøgle er spændende. Hvad er argumentet for også at have `e["uuid"]` med? - At vi får en unik rank orden?